src/redux/Action/fetchallbuses.js

```diff
@@ -1,14 +1,101 @@
 import * as actions from './actiontypes.js'
-// import Axios from './axiosConfig'
 import axios from 'axios'
-export const fetchAllbuses = () => dispatch => {
-    return axios.get(`http://localhost:5000/api/v1/buses/`)
-    .then(res => {
-      dispatch({
-        type:actions.FETCH_ALL_BUSES ,
-        payload: res.data.buses,
-
-      })
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css"
+
+const notify = (toastMsg) => toast(toastMsg);
+
+const token = localStorage.getItem('token')
+export const fetchAllbuses = () => async (dispatch) => {
+    const res = await axios.get(`https://new-avengers-be-deploy.herokuapp.com/api/v1/buses`,{
+      headers: {
+        "Authorization": `Bearer ${token}`
       }
-    )
+    });
+  dispatch({
+    type: actions.FETCH_ALL_BUSES,
+    payload: res.data.buses.rows,
+  });
+
+  };
+
+  export const deletebus = (id) => async (dispatch) => {
+    try{
+      const res = await axios.delete(`https://new-avengers-be-deploy.herokuapp.com/api/v1/buses/${id}`,{
```

Comment:
Please, make it a point to read the backend url dynamically from the environment. I want to test with my local backend